packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAds.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { MyAdsTable } from './MyAdsTable';
+
+const MyAds = () => {
+    //TODO: add empty state
+    return (
+        <div style={{ height: '100%' }}>
```

Comment:
can we use tailwind here
```suggestion
        <div className='h-full'>
```